clean_ceph_vols.py

```diff
@@ -64,22 +63,17 @@ def volume_cleanup(volume, driver):
     """
     print("Removing volume", volume.name)
     errors = {}
-    timeout = datetime.timedelta(seconds=200)
-    starttime = datetime.datetime.now()
-    while True:
-        try:
-            volobj = driver.ex_get_volume(volume.id)
-            driver.detach_volume(volobj)
-            driver.destroy_volume(volobj)
-            break
-        except BaseHTTPError as e:
-            print(e)
-            errors.update({volume.name: e.message})
-            if errors:
-                if datetime.datetime.now() - starttime > timeout:
-                    for vol, err in errors.items():
-                        print("Error destroying", vol, ":", err)
-                    return 1
+    try:
+        volobj = driver.ex_get_volume(volume.id)
+        driver.detach_volume(volobj)
+        driver.destroy_volume(volobj)
+    except BaseHTTPError as e:
+        print(e)
+        errors.update({volume.name: e.message})
+        if errors:
+            for vol, err in errors.items():
+                print("Error destroying", vol, ":", err)
+            return 1
```

Comment:
It would be good if we have consistent return types.

Reply:
Noted. Will change accordingly.